sklearn/calibration.py

```diff
@@ -62,6 +62,12 @@
     _routing_enabled,
 )
 
+__all__ = ["CalibratedClassifierCV", "calibration_curve"]
+
+
+def __dir__():
+    return __all__
```

Comment:
we should move all our implementations from root, like this one, `pipeline.py`, etc, to a subfolder (probably different PR)